desktop/apps/auction_support/client/registration_form.coffee

```diff
@@ -91,6 +91,8 @@ module.exports = class RegistrationForm extends ErrorHandlingForm
               reject "Registration submission error: #{xhr.responseJSON?.message}"
     .then (bidder) =>
       analyticsHooks.trigger 'registration:success', bidder_id: bidder.id
+      @undelegateEvents()
```

Comment:
Yes these are good questions! I'll check, but if the registration is successful then I feel like the form submission events are no longer needed. OTOH i'm not crazy about how i've peppered hiding and disabling through multiple Views and files.